lesson24/register.html

```diff
@@ -8,35 +8,217 @@
   <link rel="stylesheet" href="css/reset.css" type="text/css" />
   <link rel="stylesheet" href="css/style.css" type="text/css" />
   <title>TerraceTechフロントエンドエンジニア養成所</title>
-  <link rel="stylesheet" href="/style.css">
 </head>
 
 <body>
   <section class="membership-register">
-    <h1>会員登録</h1>
-    <form id="js-form" class="modal__form txt-center">
-      <ul>
-        <li class="form-item">
-          <label>※ユーザー名</label>
-          <input type="text">
-        </li>
-        <li class="form-item">
-          <label>※メールアドレス</label>
-          <input type="email">
-        </li>
-        <li class="form-item">
-          <label>※パスワード</label>
-          <input type="password">
-        </li>
-      </ul>
-      <div>
-        <input type="checkbox" name="check">
-        <p><a href="#">利用規約</a>を読み同意しました</p>
-        <input type="submit">送信</input>
+    <h1 class="title">会員登録</h1>
+    <form id="js-form" class="register-form">
+      <dl class="register-form__entries">
+        <div class="register-form__item">
+          <dt class="register-form__label">
+            <label for="user-name">※ユーザー名</label>
+          </dt>
+          <dd>
+            <input type="text" name="user-name" id="user-name" />
+          </dd>
+        </div>
+        <div class="register-form__item">
+          <dt class="register-form__label">
+            <label for="email">※メールアドレス</label>
+          </dt>
+          <dd>
+            <input type="email" name="email" id="email" />
+          </dd>
+        </div>
+        <div class="register-form__item">
+          <dt class="register-form__label">
+            <label for="pass">※パスワード</label>
+          </dt>
+          <dd>
+            <input type="password" name="pass" id="pass" />
+          </dd>
+        </div>
+      </dl>
+      <div class="register-form__terms">
+        <input type="checkbox" name="check" id="js-checkBox" disabled />
+        <label for="js-checkBox">
+          <a href="#" id="js-openModal" class="terms-link">利用規約</a>を読み同意しました
+        </label>
+      </div>
+      <div class="register-form__btn">
+        <input type="submit" value="送信" id="js-submitBtn" class="submit-btn btn"></input>
```

Comment:
I'd like to inform you that [Nu Html Checker](https://validator.w3.org/#validate_by_input) was giving me an error due to a single closing tag in the html <input>.

Sorry I can't review the js.
The BEM classing was so beautiful and informative!

Reply:
Thank you for telling me about this web site. I checked my coding with it and fixed.
It really helped! I appreciated you :)
[ef300f0](https://github.com/yuka830/js-lessons/pull/29/commits/ef300f0d03b251b2a439b189e3165c964f4a7d12)